pkgs/applications/science/astronomy/gprojector/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv
+, lib
+, fetchzip
+, jre
+, makeDesktopItem
+, copyDesktopItems
+, makeWrapper
+, extraJavaArgs ? "-Xms512M -Xmx2000M"
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```

If this is just wrapping gprojector can this use `stdenvNoCC`?

---

Yep, you're absolutely right, thanks for the suggestion!